Teamwork/mobile-design-system
# Update style-dictionary to 3.7.1

## This PR adds

* `launch.json` so you can run the build from inside VS Code (F5 to run it on mine - might be a different shortcut for you! :D )
* Updates style-dictionary to 3.7.1: I don't really understand the versioning for `package.json` but I needed to update to this version to get the `compose/object` format to be recognised.

## Steps to test

* `npm install` to update the dependencies
* I also ran `npm ci` - not sure if that's needed, but it was in the README so I ran that as well
* `npm run build` (or the fancy new F5! ) and check that I didn't break any iOS outputs

## More info

Updating this caused loooooads (like 3k!) of changes in `node_modules` - I'm pretty sure I need to commit these as well? I can commit them after you've reviewed anyway to make it clearer! 😄
